phnt/include/ntldr.h

```diff
@@ -852,28 +937,26 @@ NTAPI
 LdrAccessResource(
     _In_ PVOID DllHandle,
     _In_ PIMAGE_RESOURCE_DATA_ENTRY ResourceDataEntry,
-    _Out_opt_ PVOID *ResourceBuffer,
-    _Out_opt_ ULONG *ResourceLength
+    _Out_opt_ PVOID* Address,
+    _Out_opt_ PULONG Size
```

Comment:
I get that this leaked in NT5, but I find this to be a regression in clarity. :shrug: